files/task_helper.rb

```diff
@@ -0,0 +1,77 @@
+require 'json'
```

Comment:
This file should already be included based on dependencies in `metadata.json`. I'm guessing you ran into this issue testing locally? You'll want to run the same command we run in CI to download the `ruby_task_helper` to the right place:
```
git clone https://github.com/puppetlabs/puppetlabs-ruby_task_helper ../ruby_task_helper
```

We've ticketed adding `CONTRIBUTING.md` files to our modules so this is less confusing in the future, sorry you have to be the guinea pig!

Reply:
Other than that this looks good to me!

Reply:
Ah yes, then that makes sense. I figured that it would be nice to keep the tests runnable without needing anything outside of the repo, however that is a better way of achieving that rather than vendoring the helper script. Fixed now! :smile: